test/files/neg/t7299.check

```diff
@@ -1,7 +1,7 @@
-t7299.scala:4: error: implementation restricts functions to 22 parameters
+t7299.scala:4: error: functions may not have more than 22 parameters, but 23 given
```

Comment:
I wonder if we could nuance this to not say "given", since the user didn't explicitly provide them.

```
error: functions may not have more than 22 parameters, but method f has 23
```
?

You could make a new error in `ContextErrors` and then you wouldn't need to change `checkArity`, I think.